scene/gui/range.cpp

```diff
@@ -135,10 +135,6 @@ void Range::set_value(double p_val) {
 }
 
 void Range::_set_value_no_signal(double p_val) {
-	if (!Math::is_finite(p_val)) {
-		return;
-	}
-
```

Comment:
Is anything stopping this from being its own PR? I think we can address aupporting non-finite values on inspector in isolation, and not have to lump that in with serialization

Reply:
This was the original justification for the PR, but I agree it can be separate. I've removed it from this PR.